src/CoreShop/Payum/StripeBundle/Provider/LineItemProvider.php

```diff
@@ -38,16 +39,34 @@ public function getLineItem(OrderItemInterface $orderItem): ?array
         /** @var OrderInterface|null $order */
         $order = $orderItem->getOrder();
 
-        if (null === $order || $orderItem->getItemPrice() < 1) {
+        if (null === $order) {
+            return null;
+        }
+
+        $itemAmount = $this->getLineItemAmount($orderItem);
+
+        if ($itemAmount < 1) {
             return null;
         }
 
         return [
-            'amount' => $orderItem->getItemPrice(),
+            'amount' => $itemAmount,
             'currency' => $order->getCurrency()->getIsoCode(),
             'name' => $this->lineItemNameProvider->getItemName($orderItem),
-            'quantity' => $orderItem->getQuantity(),
+            'quantity' => 1,
             'images' => $this->lineItemImagesProvider->getImageUrls($orderItem),
         ];
     }
+
+    private function getLineItemAmount(OrderItemInterface $orderItem): int
+    {
+        $totalCartPriceRuleAdjustments = 0;
+        foreach ($orderItem->getAdjustments(AdjustmentInterface::CART_PRICE_RULE) as $adjustment) {
+            if ($adjustment->getNeutral()) {
```

Comment:
Not sure whether to just take "neutral" adjustments
Cart price rules are always "neutral"? @dpfaffenbauer

Reply:
usually they are. might be changed in the future though...

Reply:
So the check for "neutral" doesn't hurt?
Otherwise if adjustment is not "neutral" is included in the total Item price
Right?

Reply:
yes